### ISSUE TITLE
Connection verification sends "HOME" command. 

### ISSUE THREAD

**Issue by rewolff** [mariogrip/octodroid#23]
This morning I tried gettiing Octodroid setup to monitor the progress of my print.... 

Then after getting success for the first JSON request, it goes on to: "did this home your printer?"... 

my printer was printing  at the time. I was NOT AMUSED  that it tried to home my printer. I don't care that octoprint handled it correctly by giving priority to the running printjob. 

Please just "ask the printer for the status", and when it's idle, say: your printer seems to be idle, may I try to home it? When it's printing, just say: 'your printer seems to be printing printing. Correct?" (is there an innocent command to try in that case?)  I'm assuming you're verifying "write access" to the printer.

**Comment by mariogrip** [comment]
Right, I am checking for write access. but I can add an option to skip it. and maybe be more clear that it will try "HOME".
